packages/components/src/DateTimePickers/InputTimePicker/InputTimePicker.component.test.js

```diff
@@ -72,6 +72,9 @@ describe('InputTimePicker', () => {
 				.first()
 				.simulate('click');
 
+			jest.runAllTimers();
+
+			wrapper.update();
```

Comment:
Wondering why we need that now but not before ?

Reply:
Indeed, they're not necessary. removed.